rpmlint/checks/BinariesCheck.py

```diff
@@ -25,6 +25,48 @@ def create_nonlibc_regexp_call(call):
     r = r'.*?\s+UND\s+(%s)\s?.*$' % call
     return re.compile(r)
 
+class Issue:
```

Comment:
This should not be needed if you right away use the add_info function. It is buffered so you need not to store it for keeping I guess.